Automattic/wp-calypso
# chore: use older site for `gutenbergSimpleSiteUser` that supports CoBlocks.

#### Proposed Changes

This PR changes the default test site for `gutenbergSimpleSiteUser` to an older site (created in 2016) to support the use of CoBlocks.

#### Testing Instructions

Ensure the following build configurations are passing:

Additionally:

1. pull the branch
2. navigate to `test/e2e`
3. decrypt the secrets
4. run `yarn jest specs/blocks/blocks__coblocks__blocks.ts`

You should see:
![image](https://user-images.githubusercontent.com/6549265/212215509-1ccf2a92-eedb-4178-a9f7-60c1fae7645c.png)

#### Pre-merge Checklist





Related to #
